qa/L0_infer/infer_test.py

```diff
@@ -48,9 +48,9 @@ def _full_exact(self, input_dtype, output0_dtype, output1_dtype,
         def _infer_exact_helper(tester, pf, tensor_shape, batch_size,
                 input_dtype, output0_dtype, output1_dtype,
                 output0_raw=True, output1_raw=True,
-                model_version=None, swap=False,
+                model_version="", swap=False,
```

Comment:
We know that for Triton the version has to be an integer. So it would probably be better to keep the interface here as an integer and just convert it to a string before using the client API.

Reply:
Fixed